libraries/chain/steem_evaluator.cpp

```diff
@@ -1697,6 +1720,10 @@ namespace golos { namespace chain {
                         cv.last_update = _db.head_block_time();
                         cv.weight = 0;
                         cv.num_changes += 1;
+
+                        if (_db.has_hardfork(STEEMIT_HARDFORK_0_19__756)) {
```

Comment:
This checking on HF19 is expensive.
Before HF19 there is no records in `delegator_vote_interest_rates`
After HF19 they should be cleared in any case.

That is why, this checking will only create additional load on replay without useful work.